municipal_finance/settings.py

```diff
@@ -362,7 +361,8 @@
     "queue_limit": 100,
     "bulk": 50,
     "orm": "default",
-    "poll": 10,  # Check for queued tasks this frequently (seconds)
+    "poll": 10,
```

Comment:
why drop the doc?

Reply:
I thought that if you'd like to understand the configuration of DjangoQ you can go to the documentation. I usually like to keep comments for information specific to what is being implemented, and not use it for documentation that has a place elsewhere.

Reply:
fair point - I tend to document things like this, or at least the unit, so you don't have to look that up, but I don't feel very strongly about it